lib/binance/rest/http_client.ex

```diff
@@ -97,6 +97,30 @@ defmodule Binance.Rest.HTTPClient do
     end
   end
 
+  def request_binance(url, params, method) do
```

Comment:
I think you should *not* implement a new function here but use the `unsigned_request_binance` function instead.

Docs above it need to be fixed as it will now accept get requests as well.

Then another clause to `defp do_unsigned_request(url, params, :get, headers) do` could be added where you would do the conversion to URL params.

Reply:
DONE